fallout/dataset-test-nosqlbench.yaml

```diff
@@ -0,0 +1,297 @@
+min_version: "5.17.3"
```

Comment:
what was the reason for adding a new file, why not using the one from the `nosqlbench/`? Where do they differ?

Reply:
Aha I see now that this is not only the CRUD.. I was thinking we agreed dataset tests will use more crud-like approach..

Reply:
Didn't understand this statement. Can we connect on call?